analytics/lib/counts.py

```diff
@@ -111,6 +112,17 @@ def do_delete_count_stat_at_hour(stat, end_time):
     RealmCount.objects.filter(property = stat.property, end_time = end_time).delete()
     InstallationCount.objects.filter(property = stat.property, end_time = end_time).delete()
 
+def get_group_by_arguments(stat):
+    # type: (CountStat) -> Tuple[str, str]
+    if stat.group_by is None:
+        # if none, we compose the query with inserting the default null value into the subgroup column
+        subgroup = ', NULL'
```

Comment:
sorry, should have been more clear. The suggestion is to remove the comma from both line 119 and 122, and add it to each of the count queries.